packages/pds/bin/migration-create.ts

```diff
@@ -1,38 +0,0 @@
-#!/usr/bin/env ts-node
```

Comment:
Ah, had this become outdated?

Reply:
So I didn't fix the script. I can revert that if you want.